Gruntfile.js

```diff
@@ -660,7 +660,7 @@ module.exports = function (grunt) {
         /*'cssmin',*/
         /*'uglify',*/
         'copy:dist',
-        'npm-install',
+        // 'npm-install',
```

Comment:
it's git so you can always get this line back. If this isn't a work-in-progress PR, might as well just remove the line (and the commented lines above maybe?)